v19.2/unsplit-at.md

```diff
@@ -0,0 +1,192 @@
+        31 | \277\214*2\000    | /Table/55/4/25   | \277\214*d\000    | /Table/55/4/5E+1 | movr          | rides      | revenue_idx | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            2
+        32 | \277\214*d\000    | /Table/55/4/5E+1 | \277\214*\226\000 | /Table/55/4/75   | movr          | rides      | revenue_idx | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            1
+        33 | \277\214*\226\000 | /Table/55/4/75   | \300              | /Table/56        | movr          | rides      | revenue_idx | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            1
+(5 rows)
+~~~
+
+Now unsplit the index:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER INDEX rides@revenue_idx UNSPLIT AT VALUES (25.00), (50.00), (75.00);
+~~~
+~~~
+         key        |      pretty
++-------------------+------------------+
+  \277\214*2\000    | /Table/55/4/25
+  \277\214*d\000    | /Table/55/4/5E+1
+  \277\214*\226\000 | /Table/55/4/75
+(3 rows)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM crdb_internal.ranges WHERE table_name='rides';
+~~~
+~~~
+  range_id |     start_key     |   start_pretty   |      end_key      |    end_pretty    | database_name | table_name | index_name  | replicas | learner_replicas |       split_enforced_until       | lease_holder
++----------+-------------------+------------------+-------------------+------------------+---------------+------------+-------------+----------+------------------+----------------------------------+--------------+
+        23 | \277              | /Table/55        | \277\214          | /Table/55/4      | movr          | rides      |             | {1,2,3}  | {}               | NULL                             |            2
+        30 | \277\214          | /Table/55/4      | \277\214*2\000    | /Table/55/4/25   | movr          | rides      | revenue_idx | {1,2,3}  | {}               | 2019-09-03 18:01:02.973053+00:00 |            1
+        31 | \277\214*2\000    | /Table/55/4/25   | \277\214*d\000    | /Table/55/4/5E+1 | movr          | rides      | revenue_idx | {1,2,3}  | {}               | NULL                             |            2
+        32 | \277\214*d\000    | /Table/55/4/5E+1 | \277\214*\226\000 | /Table/55/4/75   | movr          | rides      | revenue_idx | {1,2,3}  | {}               | NULL                             |            1
+        33 | \277\214*\226\000 | /Table/55/4/75   | \300              | /Table/56        | movr          | rides      | revenue_idx | {1,2,3}  | {}               | NULL                             |            1
+(5 rows)
+~~~
+
+The `split_enforced_until` column is now `NULL` for all ranges in the `rides` table. The table is still split into ranges at `25.00`, `50.00`, and `75.00`, but the split is no longer enforced.
+
+
+## See also
+
```

Comment:
I think we should also link to [the Range Merges doc](https://www.cockroachlabs.com/docs/v19.1/range-merges.html) here.